frontend/pom.xml

```diff
@@ -85,6 +85,9 @@
               <arguments>
                 <argument>${npm.cli.script}</argument>
                 <argument>install</argument>
+                <argument>--cache-min</argument>
+                <!-- in seconds -->
+                <argument>999999</argument>
```

Comment:
There is no snapshot in js dependencies. I've tried it on my local machine. It will still build some native dependencies (written in C/C++ I think) but it seems not making any internet connections any more. The only down side I can think of is, it shares with your global npm/node configuration as well as cache.